modules/php/manifests/fpm/pool.pp

```diff
@@ -54,7 +54,7 @@
         'listen.backlog' => 256,
         'pm'     => 'static',
         'pm.max_children' => $facts['virtual_processor_count'],
-        'pm.max_requests' => 1000,
+        'pm.max_requests' => 10000,
```

Comment:
Should leave this at a 1000?

Reply:
The PR just fully reverts them to what they were prior.